cli/src/test/java/com/devonfw/tools/ide/commandlet/EditionGetCommandletTest.java

```diff
@@ -90,10 +87,10 @@ public void testEditionGetCommandletInstalledEditionToolNotInstalled() {
     // act
     editionGet.run();
     // assert
-    assertThat(context).log().hasEntries(IdeLogEntry.ofWarning("No installation of tool java was found."),
-        IdeLogEntry.ofInfo("The configured edition for tool java is java"),
-        IdeLogEntry.ofInfo("To install that edition call the following command:"),
-        IdeLogEntry.ofInfo("ide install java"));
+    assertThat(context).log().hasEntries(IdeLogEntry.ofProcessable("No installation of tool java was found."),
+        IdeLogEntry.ofProcessable("The configured edition for tool java is java"),
+        IdeLogEntry.ofProcessable("To install that edition call the following command:"),
+        IdeLogEntry.ofProcessable("ide install java"));
```

Comment:
See my comment about the failing test...
```suggestion
    assertThat(context).log().hasEntries(IdeLogEntry.ofProcessable("java"));
```